apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -221,21 +209,22 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
   // Handle the case in which the modal is created with an existing value.
   useEffect(() => {
     if (mode === Mode.INITIAL) {
-      const currentValueString = currentAiModalField?.getValue();
-      if (currentValueString) {
-        const currentValue: AiFieldValue = JSON.parse(currentValueString);
+      const previousInputString = currentAiModalField?.getValue();
+      if (previousInputString) {
+        const previousInput: AiFieldValue = JSON.parse(previousInputString);
+
```

Comment:
`currentValues` is pretty vague, so I changed it to the more descriptive seeming `previousInput`.

Reply:
+1 Brendan's comment.

Reply:
Yeah, but it's also not actually a current value _in the modal_. It's the value that the modal reads from the blockly block when it's opened up, and that is then used to populate the `inputs` field of the modal, and then _that_ becomes the _actual_ current inputs value.

It's used to create inputs which are the current ones, but they haven't become current _yet_.

Reply:
What about `savedValueString`?

Reply:
how about `initialValueString`/ `initialInput`? Emphasizes more that it's the value that's going into the modal.

Reply:
SGTM!